packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -275,6 +285,13 @@ struct WindowData {
             self->m_useDirectDebugger = IsDlgButtonChecked(hwnd, IDC_DIRECTDEBUGGER) == BST_CHECKED;
             self->m_breakOnNextLine = IsDlgButtonChecked(hwnd, IDC_BREAKONNEXTLINE) == BST_CHECKED;
 
+            auto themeComboBox = GetDlgItem(hwnd, IDC_THEME);
+            self->m_theme = ComboBox_GetCurSel(themeComboBox);
+            auto theme = self->m_theme == 0 ? xaml::ElementTheme::Default
+                                            : self->m_theme == 1 ? xaml::ElementTheme::Light : xaml::ElementTheme::Dark;
```

Comment:
```suggestion
            auto theme;
            switch (self->m_theme) {
                // Matches list item order populated in IDC_THEME above
                default: theme = xaml::ElementTheme::Default; break
                case 1: theme = xaml::ElementTheme::Light; break;
                case 2: theme = xaml::ElementTheme::Dark; break;
            }
```
IMO it'd read a bit better a switch and better map the definition above. But meh.